wp-content/themes/access/views/single-page.twig

```diff
@@ -29,7 +32,7 @@
     <div class="usa-grid {% if sidebar %}screen-desktop:layout--sidebar-gutter{% endif %}">
       {% if sidebar %}
       <aside>
-        <nav class="c-side-nav sticky top-0 pt-1 print:hidden">
+        <nav class="c-side-nav sticky top-0 pt-1 print:hidden" itemprop="breadcrumb">
```

Comment:
Bread crumb would refer to the hierarchy of the page itself so it doesn't really apply to the jump nav. This can be removed.